pom.xml

```diff
@@ -53,6 +53,11 @@
 			<scope>runtime</scope>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
```

Comment:
La dipendenza da hibernate-envers non è necessaria perché già presente in spring-data-envers